tests/read_config_test.py

```diff
@@ -37,18 +38,47 @@ def test_invalid(self):
             oac.load_config((REPO_PATH + TOML_INVALID_NAME))
 
 
+@pytest.fixture(name="setup_arguments", scope="class")
+def fixture_setup_arguments():
+    """Setup arguments for check_config
+
+    Returns:
+        dict, dict: Configuration template and default config
+    """
+    config_template = {
+        "species": {"inv": Iterable, "out": Iterable},
+        "inventories": {"dir": str, "files": Iterable},
```

Comment:
Here we also need to add `inventories.rel_to_base`

Reply:
added